### PR TITLE
uniformed soldier zombies for military bases

### PR DESCRIPTION
**Pull request by SenpaiSlime**
#### Summary
Content "Added basic zombies in military fatigues to military bases"

#### Purpose of change
Military bases at the moment have no soldier zombies that arent fully decked in gear, despite bases commonly having soldiers doing R and R and etc there, i would like to add some fitting variety to the military base populations.

#### Describe the solution
Using copy from i have created a zombie with its own monsterdrops group to simulate the presence of military personnel that are not fully kitted to fight.

#### Testing
Ive loaded a map and killed a few dozen of them, they drop what they should, they work just fine

### CODE REVIEW

**Comment by Inglonias** [comment]
This makes military bases an easier target for looting, which may or may not be a problem. They have a lifetime supply of basically everything.

**Comment by SenpaiSlime** [comment]
> 
> 
> This makes military bases an easier target for looting, which may or may not be a problem. They have a lifetime supply of basically everything.

This is not meant to make things easier, it is technically making the target more valuable yet, but the zombies im supplementing are the basic ones, they just wear different gear. If i wanted to be evil id have them copy from soldier zombies, and remove the armor, let me know what sounds better, idk how the probabilities actually work out for loot compared to standard zombies dropping guns and such

**Comment by Inglonias** [comment]
I meant you were making the zombies easier to fight, and therefore making it easier to access the warehouse, armory, etc. The warehouse and armory have a very large supply of goodies.

File: data/json/itemgroups/Clothing_Gear/clothing.json [hunk 1]
```diff
@@ -3447,4 +3459,4 @@
       { "item": "tie_skinny", "prob": 20 }
     ]
   }
-]
+]
```
Comment (LaVeyanFiend) [inline data/json/itemgroups/Clothing_Gear/clothing.json:3462]:
```suggestion
]

```

File: data/json/monstergroups/military.json [hunk 2]
```diff
@@ -110,9 +112,10 @@
...
       { "monster": "mon_zombie_scorched", "freq": 25, "cost_multiplier": 1 },
       { "monster": "mon_zombie_soldier", "freq": 200, "cost_multiplier": 2 },
+      { "monster": "mon_zombie_milbase_personnel", "freq": 50, "cost_multiplier": 1 },
       { "monster": "mon_zombie_military_pilot", "freq": 20, "cost_multiplier": 1 },
       { "monster": "mon_zombie_flamer", "freq": 1, "cost_multiplier": 30 },
       { "monster": "mon_zombie_scientist", "freq": 100, "cost_multiplier": 1 }
     ]
   }
-]
+]
```
Comment (LaVeyanFiend) [inline data/json/monstergroups/military.json:121]:
```suggestion
]

```

File: data/json/monsters/zed_soldiers.json [hunk 3]
```diff
@@ -298,6 +298,17 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_milbase_personnel",
+    "type": "MONSTER",
+    "name": { "str": "uniformed zombie" },
+    "description": "This undead soldier is adorned in standard military fatigues. It carries itself rather steadily for a zombie.",
```
Comment (LaVeyanFiend) [inline data/json/monsters/zed_soldiers.json:305]:
```suggestion
    "description": "This undead soldier is adorned in standard military fatigues.  It carries itself rather steadily for a zombie.",
```

**Comment by krulunio** [comment]
> I meant you were making the zombies easier to fight

Then just spawn more of them, lol.

Also, may I suggest adding more specific military zeds, like radio operators or commanders? I can make them.

**Comment by SenpaiSlime** [comment]
> 
> 
> > I meant you were making the zombies easier to fight
> 
> Then just spawn more of them, lol.
> 
> Also, may I suggest adding more specific military zeds, like radio operators or commanders? I can make them.
I have officers planned when the right uniforms are added, radio operators idk, these zeds are ones at a base, not out in the field with gear

**Comment by faefux** [comment]
> > > I meant you were making the zombies easier to fight
> > 
> > 
> > Then just spawn more of them, lol.
> > Also, may I suggest adding more specific military zeds, like radio operators or commanders? I can make them.
> 
> I have officers planned when the right uniforms are added, radio operators idk, these zeds are ones at a base, not out in the field with gear, im also gonna be giving them field caps when those are added, surprised they dont exist already

What will differentiate officer or other military personnel from other blob infected humans exactly?

**Comment by Zireael07** [comment]
> What will differentiate officer or other military personnel from other blob infected humans exactly?

Gear and skills, most likely.

**Comment by faefux** [comment]
> > What will differentiate officer or other military personnel from other blob infected humans exactly?
> 
> Gear and skills, most likely.

My humble opinion is that the officer could work similar to necromancer type zombie. Maybe also evolves them? Like turning some regular meds to brutes on revival,  Or other such other support abilities.

**Comment by Zireael07** [comment]
Support abilities for officer zeds are a very good idea - maybe not outright revival/upgrade, but something similar, I don't know, a bonus to morale or to damage?

**Comment by faefux** [comment]
> Support abilities for officer zeds are a very good idea - maybe not outright revival/upgrade, but something similar, I don't know, a bonus to morale or to damage?

That's a very cool idea. Buffs in general I think would be a very neat addition that kould be used for various monsters down the road. It also allows for evolution of the monster as each evolution would simply increase the buff. Would also make late game small to medium sized hords of weaker zeds more of a threat perhaps.

**Comment by SenpaiSlime** [comment]
I have plans for different PRs that introduce new zombie types that are not typical zombies, i wanna see what i can do with the current system to make some more unique threats that fill niches

**Comment by YourCompanionCube** [comment]
i would like to point out that based on the wreck at the gate and the mass graves that the base would have been on high alert and probably not have that many people who weren't geared up and at least armed

**Comment by I-am-Erk** [comment]
> i would like to point out that based on the wreck at the gate and the mass graves that the base would have been on high alert and probably not have that many people who weren't geared up and at least armed

Depends on the order of operations. The base probably fell over a period of time, the mass graves or the wreck might be earlier on, or the uniformed zombies could have died before that stuff happened

**Pull request merged**